interface/billing/sl_eob_search.php

```diff
@@ -675,43 +676,41 @@ function toEncSummary(pid) {
         }
     </style>
     <?php
-    if ($GLOBALS['enable_help'] == 1) {
-        $help_icon = '<a class="pull-right oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="' . xla("Click to view Help") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
-    } elseif ($GLOBALS['enable_help'] == 2) {
-        $help_icon = '<a class="pull-right oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#DCD6D0 !Important" title="' . xla("Enable help in Administration > Globals > Features > Enable Help Modal") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
-    } elseif ($GLOBALS['enable_help'] == 0) {
-        $help_icon = '';
+    $arrOeUiSettings = array(
+    'heading_title' => xl('EOB Posting - Search'),
+    'include_patient_name' => false,
+    'expandable' => 1,
+    'expandable_files' => array('sl_eob_search_xpd'),//all file names need suffix _xpd
+    'action' => "reset",
+    'action_title' => "",
```

Comment:
btw, when a title is entered here, then it should be translated via xl function (note do not html escape it here though)